include/seqan3/alphabet/structure/wuss.hpp

```diff
@@ -320,13 +320,16 @@ struct wuss
         {
             std::array<unsigned char, value_size> interaction_table{};
 
-            for (rank_type rnk = UNPAIRED.to_rank(); rnk <= UNPAIRED6.to_rank(); ++rnk)
+            for (rank_type rnk = static_cast<rank_type>(internal_type::UNPAIRED);
+                 rnk <= static_cast<rank_type>(internal_type::UNPAIRED6); ++rnk)
```

Comment:
please put the `++rnk` on a new line. And for the others.

Reply:
done 👍